src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -2278,6 +2275,23 @@ public void testPreSignedUrlHeaderOverrides() throws Exception {
         Assert.assertEquals(contentDisposition, response.getHeaders().getFirst(RestUtil.HEADER_CONTENT_DISPOSITION));
     }
 
+    @Test
+    public void testVPoolHeader() {
+        CreateBucketRequest createBucketRequest = new CreateBucketRequest("looney-bucket")
+                .withVPoolId("plylab-sp1");
```

Comment:
this should be a `test.properties` entry, call it `nonDefaultVpoolId`.. and make sure it is a vpool URN that is not the default for the NS of the `s3.accessKey`.